pwa/src/layout/Head.tsx

```diff
@@ -1,12 +1,19 @@
 import * as React from "react";
```

Comment:
I think you forgot to format this file

Reply:
I also noticed that when we are on the login page the title is `Dashboard` that's fine for now, but it might be good to look at that in the future